### PR TITLE
This adds content about the GitHub Token Scanning feature.

### PR DESCRIPTION
**Pull request by petra-sargent**
This adds content about the GitHub Token Scanning feature. [SHIP-1297]

[SHIP-1297]: https://cloudbees.atlassian.net/browse/SHIP-1297

### CODE REVIEW

**Comment by petra-sargent** [comment]
* Should we include the regex in the product documentation?  I'm working on fixing the rendering problem. Thanks.

File: _pro/builds-and-configuration/environment-variables.md [hunk 1]
```diff
@@ -78,7 +78,17 @@ Navigate to _Project Settings_ > _General_ and you'll find a section labeled _AE
 
 ![AES key]({{ site.baseurl }}/images/docker/aes_key.png)
 
-Save that file as `codeship.aes` in your repository root and don't forget to add the key to your `.gitignore` file so you don't accidentally commit it to your repository.
+Save that file as `codeship.aes` in your repository root. You can add the key to your `.gitignore` file so you don't accidentally commit it to your repository.
+
+The GitHub Token Scanning feature makes sure that the file`codeship.aes` is not accidentally compromised and committed to the repositories. See [Token Scanning](https://developer.github.com/partnerships/token-scanning/) for more information.
```
Comment (joshpuetz) [inline _pro/builds-and-configuration/environment-variables.md]: This is KINDA accurate: what the GitHub token scanning service is doing is looking for token names that match the regexes. It actually doesn't care what the file name is: it's looking for these tokens in ANY file of their commit (and only for public repos!)

If a user were to commit an unencrypted `codeship.aes` file to a publicly available repo, the scanning service would probably pick it up via a regex match on one of patterns listed below. Probably, but not guaranteed!

Comment (joshpuetz) [reply _pro/builds-and-configuration/environment-variables.md]: Awesome, that sounds great!

Comment (petra-sargent) [reply _pro/builds-and-configuration/environment-variables.md]: Thank you. Updated the content!

**Pull request merged**